src/wallet/coinselection.cpp

```diff
@@ -87,13 +87,18 @@ std::optional<SelectionResult> SelectCoinsBnB(std::vector<OutputGroup>& utxo_poo
     std::vector<size_t> best_selection;
     CAmount best_waste = MAX_MONEY;
 
+    // Check if the current feerate is expensive.  If it is expensive, we minimize the size of
+    // the result set and thus minimizing the transaction fee.  If the fees are not expensive,
+    // we maximize the result set to reduce the amount of UTXOs (fragmentation).
+    bool is_feerate_expensive = utxo_pool.at(0).fee > utxo_pool.at(0).long_term_fee;
```

Comment:
My thought process was simply that we are checking if the current fee rate is expensive compared to the longterm fee rate.  The reason I like expensive is it's a monetary term and "high" can pertain to many things.  Since you feel strongly about using "high" I'll change it to your suggestion.

Reply:
A non-representative survey among my colleagues resulted in three out of three (including one native English speaker with a finance background) agreeing with me: a transaction is expensive when its feerate is high.

I therefore maintain that feerates are “high” or “low”, not “expensive” or “cheap”.

Reply:
It occurred to me that in the study of computation, expensive is sometimes used to describe a computation that takes a lot of resource, therefore it's not purely a finical term.  Also, since we are terming the transaction as expensive, it does make sense to term the feerate as high.